evap/evaluation/models.py

```diff
@@ -186,12 +186,16 @@ def save(self, *args, **kw):
         if not self.general_contribution:
             self.contributions.create(contributor=None)
 
-    def is_fully_checked(self):
-        """Shortcut for finding out whether all text answers to this course have been checked"""
+    def is_fully_reviewed(self):
+        """Shortcut for finding out whether all text answers to this course have been reviewed"""
         return not self.open_textanswer_set.exists()
 
-    def is_fully_checked_except(self, ignored_answers):
-        """Shortcut for finding out if all text answers to this course have been checked except for specified answers"""
+    def is_not_fully_reviewed(self):
+        """Shortcut for finding out whether some text answers to this course have not yet been reviewed"""
+        return self.open_textanswer_set.exists()
+
+    def is_fully_reviewed_except(self, ignored_answers):
+        """Shortcut for finding out if all text answers to this course have been reviewed except for specified answers"""
```

Comment:
comments will be removed completely (it's really not hard to see what is done there) and is_fully_reviewed_except wasn't used anywhere so i'll remove the method.